.github/workflows/team.yml

```
@@ -72,7 +72,7 @@ jobs:
           author: ${{ steps.user.outputs.git-string }}
           committer: ${{ steps.user.outputs.git-string }}
           commit-message: "maintainers/github-teams.json: Automated sync"
-          branch: github-team-sync
+          branch: create-pull-request/github-team-sync
```

I implemented it as this:

```suggestion
          branch: pr/github-team-sync
```